ui/src/components/Form/ModalFormErrorHandled.tsx

```diff
@@ -17,13 +17,15 @@ type Props = {
   children: Callable | Renderable;
   onSubmit: () => Promise<void>;
   disabled?: boolean;
+  button?: boolean;
 };
 
 const ModalFormErrorHandled: (props: Props) => React.ReactElement = ({
   title,
   children,
   onSubmit,
   disabled,
+  button,
```

Comment:
i feel like this boolean name could be slightly more descriptive? `hasButton` or `clickableTitle` ? I wasnt sure at first what `button` was for